### ISSUE TITLE
FilePicker | Support for @bind-Files

### ISSUE THREAD

**Issue by David-Moreira** [Megabit/Blazorise#5796]
Investigate whether it would be feasible to add FileEntry programmatically to the Files collection and making this as a bindable Parameter. This would allow the user to populate with entries.
And whether there would be any limitations. For example, these entries not having file handles, since they are created programmatically in c#

**Comment by Hyppothalamus** [comment]
Is there any progress on this issue? This would be something I would like to see.

**Comment by stsrki** [comment]
I don't think this is technically possible at all. For security purposes, files can only be picked by the browser and only when the user manually click on the file input. Am I wrong, or is there some new API that we could use?

**Comment by Hyppothalamus** [comment]
I would not really need "native" files. For my use case I would just need to be able to set in the filepicker for example a default value for a previous uploaded file so the user could edit the file if needed to. Not really setting files to be uploaded, if that makes sense.

**Comment by David-Moreira** [comment]
> I don't think this is technically possible at all. For security purposes, files can only be picked by the browser and only when the user manually click on the file input. Am I wrong, or is there some new API that we could use?

Chiming in on this, yes probably there would be the technicaly limitation regarding actual file handles : 
> For example, these entries not having file handles

The purpose of this would be to help the developer present files for UI purposes, i.e
**File 1 - Uploaded
File 2 - Uploaded
File 3 - Error (please reupload)**
Generally when you want to have persistent state, and when the end user comes back to the page he's able to see a list of the files
